common/src/Microsoft.Azure.IIoT.Core/src/Hub/Services/IoTHubEdgeBaseDeployment.cs

```diff
@@ -21,6 +21,12 @@ public sealed class IoTHubEdgeBaseDeployment : IHostProcess {
         public static readonly string TargetCondition =
             $"(tags.__type__ = '{IdentityType.Gateway}' AND NOT IS_DEFINED(tags.unmanaged))";
 
+        /// <summary>
+        /// Target condition for standalone publisher
+        /// </summary>
+        public static readonly string TargetConditionStandalone =
+            $"(tags.__type__ = '{IdentityType.Gateway}' AND tags.unmanage = 'true')";
```

Comment:
Used for testing only, please move to e2e-tests/IoTPlatform-E2E-Tests solution

Reply:
Using layered deployment for `tags.unmanaged = 'true'` case defeats the purpose device being unmanaged. Maybe we should create module definitions directly for the IoT Edge device instead of using a layered deployment.

Reply:
moved

Reply:
@dacolgit, please remove `TargetConditionStandalone` from `IoTHubEdgeBaseDeployment.cs` as it is not used.